.gitignore

```diff
@@ -2,4 +2,6 @@ node_modules/
 bower_components/
 config/config.js
 rise-financial-version.js
+rise-financial.html
```

Comment:
@stulees This is akin to when we were committing the `dist` folder, which created problems for us when we weren't even sure it was necessary. I think we're in a similar situation here. If a designer is already pulling this component in via Bower or similar, then I tend to think that it's not a stretch for them to run `gulp build`.

Given this, what do you think about leaving it out for now and adding them back in if it becomes an issue? It creates extra work for us by committing them.